src/MainUtils.cpp

```diff
@@ -321,6 +360,7 @@ void genModelObject(const mlir::OwningModuleRef &module, string bitcodePath,
   Command llvmToObj(/*exePath=*/!llcPath.empty() ? llcPath : kLlcPath);
   llvmToObj.appendStr("-filetype=obj")
       .appendStr("-relocation-model=pic")
+      .appendStr(getTargetOptions())
```

Comment:
Any reasons that we also add "-O3" here?

Reply:
the pipeline is to call opt to optimize the llvm IR (with -O3) and subsequently call llc to take the optimized bitcode and convert to machine code .... so, we don't need -O3 on the llc patrt